css/properties/rotate.json

```diff
@@ -6,10 +6,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/rotate",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "73"
```

Comment:
Either set this to false or wait three weeks to see if this feature makes it to Chrome beta. Chrome requires what's called an 'Intent to Ship' before something can go out the door. I cannot find one for CSS rotate.

Reply:
Hey @jpmedley, was there any movement on this? I'm just checking to see what @estelle should up date this to, so we can get this data merged. Cheers!

Reply:
I didn't look at this as closely as I should have last time. I think `true` is the best that can be done. Because of file name changes, the commit history cuts off after 18 months. I have no way of know how much older than that it is.

Reply:
OK, true it is then. @estelle, can you updates these please?

Reply:
changed all 73s to true